cmdk/src/index.tsx

```diff
@@ -14,7 +14,7 @@ type SeparatorProps = DivProps & {
   /** Whether this separator should always be rendered. Useful if you disable automatic filtering. */
   alwaysRender?: boolean
 }
-type DialogProps = RadixDialog.DialogProps & CommandProps
+type DialogProps = RadixDialog.DialogProps  & CommandProps & { container?: HTMLElement }
```

Comment:
Would you add a JSDoc comment here describing the behavior?

```
/** Provide a custom element the Dialog should portal into */
```

Reply:
added note!